docs: temp (#4233)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/Upload/info.mdx

---
showTabs: true
---

## Description

> `Value.Upload` is a wrapper component for displaying Upload files, with user experience tailored for an array of files.

Reviewer: ```suggestion
`Value.Upload` is a value component for displaying a list of files.
```

Author: Updated :)

> `Value.Upload` is a wrapper component for displaying Upload files, with user experience tailored for an array of values.

Author: ```suggestion
`Value.Upload` is a wrapper component for displaying Upload files, with user experience tailored for an array of files.
```
